package.json

```diff
@@ -6,6 +6,6 @@
   "author": "Loskir",
   "license": "MIT",
   "dependencies": {
-    "telegraf": "^3.36.0"
+    "telegraf": ">= 3.36.0"
```

Comment:
I suggest using something like `^3.36.0||^4.0.0`

Reply:
Honestly it should be a peer dependency instead

Reply:
And what about yarn.lock, is it not required in this case?